### PR TITLE
Wrote the home page

### PR DESCRIPTION
**Pull request by groksrc**
Please fix typos and comment for changes. The more the merrier.

### CODE REVIEW

File: src/components/Home.vue [hunk 1]
```diff
@@ -1,14 +1,132 @@
...
+    h1.heading-2 Introduction
+    p.
+      Welcome! We're glad you've found us. This little library of components started at a time
+      when there were already a few choices for Vue.js component libraries, but none of them really felt
+      right to us. We wanted something lightweight, easy to get started with, and something that didn't require
+      changes to our existing apps. We also felt like composability should really be favored over configuration
+      and that wasn't as well represented as we had hoped.
+    p.
+      We've looked deeply at Element, Quasar, Blueprint (React) and others in order to get a decent idea about
+      what the community is looking for in a component library and what this library should be about. That said, we may
```
Comment (ruffin) [inline src/components/Home.vue]: -may

File: src/components/Home.vue [hunk 2]
```diff
@@ -1,14 +1,132 @@
...
+    p.
+      Components are designed towards composability over configuration. Each component should be focused solving a small
+      distinct problem. Advanced components may be introduced in the future that apply the principle of composability to
+      achieve more complex behaviors, but for now the idea is: "Do one thing, and do it well."
+
+    h1.heading-3 Naked Components
+    p.
+      Several of the libraries we looked at were themeable. We really liked that about them. That said, when we dug in
+      to the code we found several edges and gotchas where there were problems. Inline styling, lack of thought around
+      the specifc markup, and scoped styling buried in single file components are all things that we found problematic.
```
Comment (ruffin) [inline src/components/Home.vue]: specific

File: src/components/Home.vue [hunk 3]
```diff
@@ -1,14 +1,132 @@
...
+
+    h1.heading-3 Themeability
+    p.
+      The HTML/Javascript and CSS are built and maintained separately. (Yes, there are two separate builds.) The
+      stylesheet is then applied to the components by way of CSS class names. That is the only touchpoint. The benefit
+      is that we have to do the same work you have to do if you were to style the components from scratch. It makes us
+      really think about what we're doing, but it also makes theming a lot easier.
+    p.
+      Components do assume certain classes will be made available by the stylesheet of your app in order to look right.
+      But "right" by our definition and "right" by yours are probably different. So what we do is allow you to interject
```
Comment (ruffin) [inline src/components/Home.vue]: is `interject` the right word here?

File: src/components/Home.vue [hunk 4]
```diff
@@ -1,14 +1,132 @@
...
+    h1.heading-3 Plug and Play
+    p.
+      You shouldn't have to do anything special in order to use the components (such as wrap your app
+      in a special tag). Using the components should only require you to install them as a plugin via:
+    .box.box-neutral.box-code
+      pre
+        code.
+          // main.js
+          import Vueable from 'vueable'
+          import * as components from 'vueable' // or individually as you wish
```
Comment (tenkiller) [inline src/components/Home.vue]: I think you can shorten this to one line.
```javascript
import Vueable, * as components from 'vueable'
```

File: src/components/Home.vue [hunk 5]
```diff
@@ -1,14 +1,132 @@
...
+    p.
+      To get going with Vueable, all you need to do is import the library and include the stylesheet:
+    .box.box-neutral.box-code
+      pre
+        code.
+          // main.js
+          import 'vueable/dist/production/default.css' // or include me in the style tag of your App.vue
+          import Vue from 'vue'
+          import Vueable from 'vueable'
+          import * as components from 'vueable'
```
Comment (tenkiller) [inline src/components/Home.vue]: Same here. I think you can shorten this to one line.
```javascript
import Vueable, * as components from 'vueable'
```

File: src/components/Home.vue [hunk 6]
```diff
@@ -1,14 +1,130 @@
...
+      Welcome! We're glad you've found us. This little library of components started at a time
+      when there were already a few choices for Vue.js component libraries, but none of them really felt
+      right to us. We wanted something lightweight, easy to get started with, and something that didn't require
+      changes to our existing apps. We also felt like composability should really be favored over configuration
+      and that wasn't as well represented as we had hoped.
+    p.
+      We've looked deeply at Element, Quasar, Blueprint (React) and others in order to get a decent idea about
+      what the community is looking for in a component library and what this library should be about. That said, we
+      are not terribly original in our thinking or comprehensive in our component list. But our hope is that if
+      you need a basic set of components to do simple UIs with then we might be a fit for you.
```
Comment (ruffin) [inline src/components/Home.vue]: *puts on editor hat*

This is a really good start. I like the layout and the concepts presented under headings, so people really get the philosophy behind why Vueable is useful.

I'm not sure about the "we" concept for these first two paragraphs. I would write it in a more declarative style, about the library itself, rather than introducing it as "we made this, here's the story". Something like:

Vueable is a lightweight component library designed to be composable, simple to configure, and easy to integrate into existing apps.

It takes inspiration from Element, Quasar, and Blueprint (React). As a work in progress, it is not comprehensive or even terribly original in its component list. But if you need a basic set of components for simple UIs, Vueable may be a good fit.

Comment (ruffin) [reply src/components/Home.vue]: I have a really terse writing style, so feel free to ignore this. I've been told I'm kind of a jerk for cutting a lot of words :)

**Comment by ruffin** [comment]
:shipit:
